src/Enterspeed.Source.UmbracoCms.V8/Services/EnterspeedPropertyService.cs

```diff
@@ -105,9 +105,19 @@ private IEnterspeedProperty CreateMetaData(IPublishedContent content, string cul
                 ["nodePath"] = new ArrayEnterspeedProperty("nodePath", GetNodePath(content.Path, culture))
             };
 
+            MapAdditionalMetaData(metaData);
+
             return new ObjectEnterspeedProperty("metaData", metaData);
         }
 
+        /// <summary>
+        /// Override to add extra meta data
+        /// </summary>
+        /// <param name="metaData"></param>
+        protected virtual void MapAdditionalMetaData(Dictionary<string, IEnterspeedProperty> metaData)
```

Comment:
I don't know the full use case for this. I was just wondering if we should pass the IPublishedContent object to this method, so users can map potential additional meta data from the content node?

Reply:
Maybe also pass the culture?

Reply:
That is a good idea. I will implement it.